### PR TITLE
Update dockerfile for core 2.67.1

### PR DESCRIPTION
**Pull request by joschrew**
includes: 
- update version of ocrd-core base image to latest core version
- fix work/build-dir usage and removal. Build dir in core was changed to /build/core
- remove ocrd-network logfiles, which are created through resmgr call

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM ocrd/core:v2.62.0 AS base
+FROM ocrd/core:v2.67.1 AS base
```
Comment (stweil) [inline Dockerfile:1]:
```suggestion
FROM docker.io/ocrd/core:v2.67.1 AS base
```

Comment (stweil) [reply Dockerfile:1]: While updating the version, support for podman can be added, too.

Comment (bertsky) [reply Dockerfile:1]: Until now we always use Dockerhub registry implicitly, so supporting Podman would require similar changes in lots of places. Also, we don't know of Dockerhub will stay free. Therefore I'd prefer a more general mechanism.

How about generating derived Dockerfiles from the existing, git-controlled Dockerfile via makefile rule? For example, we could have:

```make

Dockerfile.podman: Dockerfile
	sed "s,FROM ocrd/core,FROM docker.io/ocrd/core," < $< > $@
	#sed "s,FROM ocrd/core,FROM ghcr.io/ocr-d/core," < $< > $@

podman: Dockerfile.podman
	podman build -f $< .
```

**Review by bertsky — APPROVED** [review]
LGTM – thx!

File: Dockerfile (hunk 1 above)
Comment (stweil) [reply Dockerfile:1]: The suggested change would not change the status quo for Docker users, but add support for Podman here without adding complexity. ocrd_olena already uses this pattern, 13 other Dockerfiles would still need it.

Comment (bertsky) [reply Dockerfile:1]: Sure, we can also do that change along the way (and come back to a more general mechanism when really needed).

**Review by stweil — APPROVED** [review]
Thanks!

File: Dockerfile [hunk 2]
```diff
@@ -1,4 +1,4 @@
-FROM ocrd/core:v2.62.0 AS base
+FROM docker.io/ocrd/core:v2.67.1 AS base
```
Comment (bertsky) [inline Dockerfile:1]:
```suggestion
FROM docker.io/ocrd/core:v2.67.2 AS base
```

**Pull request merged**